src/RelaxedRequestMatcher.php

```diff
@@ -17,10 +17,14 @@ class RelaxedRequestMatcher
 
     public static function configureOptions(array $options)
     {
-        self::$options = array_merge_recursive(array(
-            'ignoreUrlParameters' => array(),
-            'ignoreHeaders' => array(),
-        ), $options);
+        self::$options = array_merge_recursive(
+            array(
+                'ignoreUrlParameters' => array(),
+                'ignoreHeaders'       => array(),
+                'bodyScrubber'        => array(),
```

Comment:
Since this accepts an array of body scrubbers, I'd opt towards naming it `bodyScrubbers` throughout the classes instead.